layer/layer.go

```diff
@@ -199,11 +199,11 @@ func createChainIDFromParent(parent ChainID, dgsts ...DiffID) ChainID {
 		return parent
 	}
 	if parent == "" {
-		return createChainIDFromParent(ChainID(dgsts[0]), dgsts[1:]...)
+		return createChainIDFromParent(ChainID(dgsts[0]), dgsts[1:]...) //nolint:gosec // FIXME(thaJeztah): ignore G602: slice index out of range, which is a false positive
```

Comment:
Use nosec comments so that other lint errors from gosec are still reported for the line.
```suggestion
		return createChainIDFromParent(ChainID(dgsts[0]), dgsts[1:]...) // #nosec G602 -- slice index out of range, which is a false positive
```